code/datums/supplypacks/security.dm

```diff
@@ -167,6 +167,16 @@
 	containername = "Security surplus equipment"
 	access = null
 
+/datum/supply_packs/security/camerapods
```

Comment:
Let's remove this, for now.

Reply:
Just to check, you know this means they won't be available outside of admins spawning them in?

Reply:
Yes, that's the idea.